include/json/value.h

```diff
@@ -463,8 +463,9 @@ Json::Value obj_value(Json::objectValue); // {}
   ///
   /// Equivalent to jsonvalue[jsonvalue.size()] = value;
   Value& append(const Value& value);
+  /// \brief Insert value in array at specific index
+  bool insert(ArrayIndex index, const Value& newValue);
```

Comment:
A movable `insert` method should also be provided, which may declare as

```
bool insert(ArrayIndex index, Value&& newValue);
```

Reply:
Thank you for your advice. I'm not sure if it's necessary to use move assignment operator here.

Reply:
Thank you gitpaladin, I'm gonna to add this today.